SavedInstances/Core/Core.lua

```diff
@@ -1621,6 +1624,7 @@ hoverTooltip.ShowToonTooltip = function (cell, arg, ...)
     indicatortip:AddLine(COMBAT_XP_GAIN, format("%.0f%% + %.0f%%", t.XP / t.MaxXP * 100, percent))
   end
   indicatortip:AddLine(STAT_AVERAGE_ITEM_LEVEL,("%d "):format(t.IL or 0)..STAT_AVERAGE_ITEM_LEVEL_EQUIPPED:format(t.ILe or 0))
+  indicatortip:AddLine("PvP Item Level",("%d"):format(t.ILPvp or 0))
```

Comment:
Use global string LFG_LIST_ITEM_LEVEL_INSTR_PVP_SHORT to replace "PvP Item Level"

Reply:
Thanks, where do you find this list of global strings? I'm new to wow addon development.

This has been updated.